### DISCUSSION TITLE
E3v2 DWIN Display keeps crashing

### DISCUSSION THREAD

**Discussion by LegenDarius01** [mriscoc/Ender3V2S1#495] [Q&A]
Hi, 
First of all: Love your work!
However, Ive used ur Firmwares for quite a while now. Recently I updated to the newest Version of your Firmware for the E3v2 with the 4.2.2 slient board + BLTouch. Before flashing the Printer I did some changes regarding my PID settings etc.. So nothing really important regarding the actual firmware. 
And since then every second time I turn on the printer my screen simply stays black. It pressing the button or restarting doesn't help. 
I have to flash it with the original Creality firmware and then reupload your firmware to get it back to working. 

Any ideas on how to fix this? 
Thanks a lot!

**Comment by mriscoc** [comment]
Please post a link to the precompiled version that you are using.

**Reply by LegenDarius01** [reply]
Thanks a lot for your reply. 
i did manage to make it work by now. Apparently I was simply stupid and mixed up your source code version and the latest marlin bugfix. That led to some issues in the long run. 

Thanks a lot though!
